src/BottomNavigation/BottomNavigation.react.js

```diff
@@ -44,20 +42,67 @@ function getStyles(props, context) {
 */
 
 class BottomNavigation extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            moveAnimated: new Animated.Value(0 - this.context.uiTheme.bottomNavigation.container.height),
```

Comment:
There should be something like this:
```
constructor(props, context) {
    super(props, context);

    .... context.uiTheme ...
}
```
**First question:**
Does it work? :) Because the bottomNavigation is StylSheet and you should use [this method](http://facebook.github.io/react-native/releases/0.39/docs/stylesheet.html#flatten) to access to `height`. I'll try it later as well.

**Second question:**
Why we need bottom navigation to start as hidden?